lib/Backend/IRBuilderAsmJs.cpp

```diff
@@ -2553,6 +2553,10 @@ IRBuilderAsmJs::BuildInt2(Js::OpCodeAsmJs newOpcode, uint32 offset, Js::RegSlot
         }
 
         break;
+    
+    case Js::OpCodeAsmJs::Eqz_Int:
+        instr = IR::Instr::New(Js::OpCode::CmEqz_I4, dstOpnd, srcOpnd, m_func);
```

Comment:
Perhaps. I'm still thinking of whether this will require a real comparison at all. It seems to be introducing an extra layer of branching since this instruction ought only be used with if, thus together they are really an if0. I think that was the CG's intention anyway.